### PR TITLE
web: fix global search and add UI tweaks

### LINKED ISSUE #2287: webui: not obvious that entries can be expanded for initial static capabilities view
Screenshot from 2024-08-13 08-26-11

should be something like

Screenshot from 2024-08-13 08-28-36

### LINKED ISSUE #2288: webui: "show capabilities by function" clicking "show column filters" does nothing
I'd expect the column filters to be displayed as per the other pages.

### PR DESCRIPTION
**Pull request by s-ff**
closes #2288, #2289, #2290 and partially fixes #2287

This PR:
- fixes global search in show capabilities by function
- introduces search column filters in show capabilities by function (off by default)
- uses a bold color separator
- reintroduces the toggle icon (`>`)

### Checklist

- [x] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: web/explorer/src/components/SettingsPanel.vue [hunk 1]
```diff
@@ -20,9 +20,9 @@
                     />
                     <label for="showLibraryRules">
                         <span v-if="libraryRuleMatchesCount > 1">
-                            Show {{ libraryRuleMatchesCount }} library rule matches
+                            Show {{ libraryRuleMatchesCount }} distinct library rules
                         </span>
-                        <span v-else>Show 1 library rule match</span>
+                        <span v-else>Show 1 distinct library rule</span>
```
Comment (mr-tz) [inline web/explorer/src/components/SettingsPanel.vue:25]: is this also shown for 0 library rule matches?

**Review by mr-tz — APPROVED** [review]
one question, but other than that LGTM!

File: web/explorer/src/components/SettingsPanel.vue (hunk 1 above)
Comment (s-ff) [reply web/explorer/src/components/SettingsPanel.vue:25]: Good catch, I will add another `v-else-if count === 0`.

Comment (s-ff) [reply web/explorer/src/components/SettingsPanel.vue]: Should now be resolved. Feel free to test with https://mandiant.github.io/capa/explorer/#/?rdoc=https://dpaste.org/Ox4tu/raw once merged.

**Pull request merged**